etc/itch.profile

```diff
@@ -22,12 +23,13 @@ nogroups
 nonewprivs
 noroot
 notv
+novideo
 protocol unix,inet,inet6,netlink
 seccomp
 shell none
 
 private-dev
 private-tmp
 
-noexec ${HOME}
+#noexec ${HOME}
```

Comment:
Looks good! I've not used itch.io either, but do we need to comment out `noexec ${HOME}`?

Reply:
Assuming its a launcher, its downloading games and needs to be able to run them.